presentation/src/main/java/com/architect/coders/mu8/utils/PermissionsChecker.kt

```diff
@@ -0,0 +1,12 @@
+package com.architect.coders.mu8.utils
+
+import android.app.Application
+import android.content.pm.PackageManager
+import androidx.core.content.ContextCompat
+
+class PermissionsChecker(private val application: Application, private val permission: String) {
```

Comment:
Why do we need this? What permission are you checking on this PR?

Reply:
Really I am not using but it can be in other PR, Deleted